docs-new/docs/graphql/core/auth/authorization/permission-rules.mdx

```diff
@@ -50,111 +52,96 @@ or the `metadata APIs for permissions <metadata_api_permission>`.
 For `select` operations or for GraphQL queries, you can configure the
 following:
 
--   `row-level-permissions`
--   `col-level-permissions`
--   `aggr-query-permissions`
--   `limit-rows-permissions`
+- [Row-level permissions](/graphql/core/auth/authorization/permission-rules.mdx#row-level-permissions)
+- [Column-level permissions](/graphql/core/auth/authorization/permission-rules.mdx#col-level-permissions)
+- [Aggregation queries permissions](/graphql/core/auth/authorization/permission-rules.mdx#aggr-query-permissions)
+- [Row fetch limit](/graphql/core/auth/authorization/permission-rules.mdx#limit-rows-permissions)
 
 ### **Insert** permissions
 
-For `insert` operations or for GraphQL mutations of the type *insert*,
+For `insert` operations or for GraphQL mutations of the type _insert_,
 you can configure the following:
 
--   `row-level-permissions`
--   `col-level-permissions`
--   `col-presets-permissions`
--   `backend-only-permissions`
+- [Row-level permissions](/graphql/core/auth/authorization/permission-rules.mdx#row-level-permissions)
+- [Column-level permissions](/graphql/core/auth/authorization/permission-rules.mdx#col-level-permissions)
+- [Column presets](/graphql/core/auth/authorization/permission-rules.mdx#col-presets-permissions)
+- [Backend only](/graphql/core/auth/authorization/permission-rules.mdx#backend-only-permissions)
```

Comment:
These links seem to point to sub sections within this file itself. Please crodd check and if thats the case, no need to add full path for link. The `#hash-param` alone is enough as its the same file.

Ex:
`[Row-level permissions](#row-level-permissions)`

Please check in live and update.

This applies for all similar usage - if the sub section is in same file, no need to add full path.